File: 2463-minimum-recolors-to-get-k-consecutive-black-blocks/2463-minimum-recolors-to-get-k-consecutive-black-blocks.java
class Solution {
    public int minimumRecolors(String blocks, int k) {
        int minW = Integer.MAX_VALUE;

        for (int i = 0; i <= blocks.length() - k; i++) {
            int wCount = 0;

            for (int j = i; j < i + k; j++) {
                if (blocks.charAt(j) == 'W') {
                    wCount++;
                }
            }
            System.out.println(wCount);
            if (wCount < minW) {
                minW = wCount;
            }
        }
        return minW;
    }
}

File: 2463-minimum-recolors-to-get-k-consecutive-black-blocks/2463-minimum-recolors-to-get-k-consecutive-black-blocks.kt
class Solution {
    fun minimumRecolors(blocks: String, k: Int): Int {
        var minW = Int.MAX_VALUE

        for(i in 0..blocks.length - k){
            var wCount = 0
            for(j in i..i + k-1){
                if (blocks[j] == 'W') {
                    wCount++
                }
            }
            minW = minOf(minW, wCount)
        }
        return minW;
    }
}